src/pyodide/BUILD.bazel

```diff
@@ -155,10 +153,6 @@ REPLACEMENTS = [
         "Date.now",
         "monotonicDateNow",
     ],
-    [
-        "reportUndefinedSymbols()",
-        "reportUndefinedSymbolsNoOp()",
```

Comment:
It turns out that `__heap_base` is only set to its correct value if we either preload at least one dynamic library (which moves the heap base) or if we call `reportUndefinedSymbols()`. This problem was masked before because we always loaded at least one dynamic library.